README.md

```diff
@@ -1,7 +1,7 @@
 # Discourse Clicable Topic Component
 This component makes the entire area of a `topic-list-item` clickable, as opposed to just the title.
 
-![Discourse Clickable Topic Preivew](readme-assets/preview.gif)
+![Discourse Clickable Topic Preivew](https://github.com/discourse/discourse-clickable-topic/assets/17474474/720acb71-b283-4a3a-ad98-39a0d502ca1e)
```

Comment:
Why this change? I think that GitHub assets URL works just because we used to have that preview.gif in this repository. I would keep it the way it was.

Reply:
I got the assets URL by uploading the GIF to a post in a PR for this repo and then copying the URL of the image. It's a known trick for hosting/embedding images in a README without cluttering the repository itself with image files that are only needed for the README. We also get to reduce the theme size by about 1.4MB, which is not a huge amount of data, but it's a nice win for no downsides.